programs/cordova/CMakeLists.txt

```diff
@@ -0,0 +1,88 @@
+
+configure_file(config.xml.in config.xml)
+configure_file(manifest.webapp.in manifest.webapp)
+
+#set(CORDOVA_FILES WebODF/config.xml)
+
+# install cordova in the top level node_modules directory
+set(CORDOVA ${CMAKE_BINARY_DIR}/node_modules/cordova/bin/cordova)
+add_custom_command(
+    OUTPUT ${CORDOVA}
+    COMMAND ${NPM} ARGS install cordova@3.5
+    DEPENDS copy_node_modules
+    WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
+)
+# create a project folder for WebODF cordova
+add_custom_command(
+    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/WebODF/hooks/README.md
+    COMMAND ${CMAKE_COMMAND} ARGS -E remove_directory ${CMAKE_CURRENT_BINARY_DIR}/WebODF
+    COMMAND ${CORDOVA} create WebODF org.webodf WebODF --copy-from ${CMAKE_CURRENT_SOURCE_DIR}/www
+    DEPENDS ${CORDOVA}
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+)
+# add the desired plugins
+add_custom_command(
+    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/WebODF/plugins/org.apache.cordova.file/plugin.xml
+    COMMAND ${CORDOVA} plugins add org.apache.cordova.file
+    COMMAND ${CORDOVA} plugins add ${CMAKE_CURRENT_SOURCE_DIR}/plugins/org.webodf.starturl
+    DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/WebODF/hooks/README.md
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/WebODF
+)
+# add the android platforms to the WebODF cordova folder
+add_custom_command(
+    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/WebODF/plugins/android.json
+    COMMAND ${CORDOVA} platform add android
+    DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/WebODF/plugins/org.apache.cordova.file/plugin.xml
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/WebODF
+)
+# add the firefoxos platforms to the WebODF cordova folder
+add_custom_command(
+    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/WebODF/plugins/firefoxos.json
+    COMMAND ${CORDOVA} platform add firefoxos
+    DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/WebODF/plugins/org.apache.cordova.file/plugin.xml
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/WebODF
+)
+add_custom_command(
+    OUTPUT WebODF/config.xml
```

Comment:
Fix has been updated to latest version of this PR:
https://github.com/kossebau/WebODF/commit/a64825508cc12f98800c25a4ec5f9ddf05d26eb5